### PR TITLE
initial commit

### PR DESCRIPTION
**Pull request by SuperHotDogCat**
Create AnimationIntegrator.py

### CODE REVIEW

File: AnimatedWordCloud/Animator/AnimationIntegrator.py [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+
+from typing import List
+from PIL import Image
+
+
+def integrate_images(
+    image_paths: List[str],
+    filename: str,
+    duration: int = 500,
+) -> None:
```
Comment (konbraphat51) [inline AnimatedWordCloud/Animator/AnimationIntegrator.py:18]: Add a simple test code like:
```python
assert integrate_images() == None
```

to check error

Comment (SuperHotDogCat) [reply AnimatedWordCloud/Animator/AnimationIntegrator.py:18]: OK

**Review by SuperHotDogCat — COMMENTED** [review]
pytest AnimationIntegrator_test.py 
============================================================================== test session starts ===============================================================================
platform darwin -- Python 3.8.8, pytest-7.4.3, pluggy-1.3.0
rootdir: /AnimatedWordCloud
collected 1 item                                                                                                                                                                 

AnimationIntegrator_test.py .                                                                                                                                              [100%]

=============================================================================== 1 passed in 0.27s ================================================================================

File: AnimatedWordCloud/Animator/AnimationIntegrator.py [hunk 2]
```diff
@@ -0,0 +1,39 @@
+# -*- coding: utf-8 -*-
+# Copyright (C) 2023 AnimatedWordCloud Project
+# https://github.com/konbraphat51/AnimatedWordCloud
+#
+# Licensed under the MIT License.
+"""
+Integrates the images into a single video (gif)
+"""
+
+from typing import List
```
Comment (konbraphat51) [inline AnimatedWordCloud/Animator/AnimationIntegrator.py:10]: __future__

File: AnimatedWordCloud/Animator/AnimationIntegrator.py [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+from PIL import Image
+
+
+def integrate_images(
+    image_paths: List[str],
+    filename: str,
+    duration: int = 500,
```
Comment (konbraphat51) [inline AnimatedWordCloud/Animator/AnimationIntegrator.py:17]: duration -> duration_per_frame

File: AnimatedWordCloud/Animator/AnimationIntegrator.py [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+    """
+    Create images of each frame
+
+    :param
+    List[str] image_paths: List of image_paths created by AnimatedWordCloud.Animator.ImageCreator.create_images.
+    str filename: output filename.
+    int duration: display time for each frame
+    :return: None
+    """
+
```
Comment (konbraphat51) [inline AnimatedWordCloud/Animator/AnimationIntegrator.py:28]: Use newly added `Config` class

Comment (konbraphat51) [reply AnimatedWordCloud/Animator/AnimationIntegrator.py:28]: Add necessary parameters in `Config` class

**Review by konbraphat51 — COMMENTED** [review]
Only one thing

**Review by konbraphat51 — APPROVED** [review]
Good

**Pull request merged**